.github/workflows/run-tests-with-coverage.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     steps:
       - uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: '8.1'
```

Comment:
as we discussed - we need to just implement the compatibility with PhpUnit10, but still use PhpUnit 9 version as an available

Reply:
@DenTray Done